integrity-backend/input-starling-capture-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-recorder.json

```diff
@@ -0,0 +1,42 @@
+{
+  "recorderMetadata": [
+    {
+      "service": "api",
+      "info": [
+        {
+          "type": "external",
```

Comment:
New type for  recorder components running outside our backend infra. They can be reported to us via API.